tests/phpunit/includes/storage/sqlstore/StatisticsCollectorTest.php

```diff
@@ -41,7 +41,7 @@ private function newInstance( $count = 55, $cacheEnabled = false, $hash = 'foo'
 			'isFixedPropertyTable' => true
 		) );
 
-		$store = $this->newMockBuilder()->newObject( 'Store', array(
+		$store = $this->newMockBuilder()->newObject( 'SMWSQLStore3', array(
```

Comment:
@mwjames do I first need to register additional  stuff in this mock builder or what?